### PR TITLE
Add webhooks and other athlete authentication to changelog

### PR DESCRIPTION
**Pull request by zackisaacs**
screen shot 2018-01-18 at 2 19 46 pm

### CODE REVIEW

File: content/docs/changelog.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The Strava mobile applications and 3rd party applications use the V3 API to communicate with Strava. The Strava API Team strives to provide a stable interface with clear documentation. It is important to maintain a clear record of functional changes to the V3 API; the changelog is the official external record of these changes.
 
+###### January 17, 2018
++ Require authentication for athlete-specific data.
```
Comment (MateoAO) [inline content/docs/changelog.md]: Athlete-specific data will requires authentication.


We can edit

Comment (MateoAO) [reply content/docs/changelog.md]: later if we

Comment (MateoAO) [reply content/docs/changelog.md]: need to

File: content/docs/changelog.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 The Strava mobile applications and 3rd party applications use the V3 API to communicate with Strava. The Strava API Team strives to provide a stable interface with clear documentation. It is important to maintain a clear record of functional changes to the V3 API; the changelog is the official external record of these changes.
 
+###### January 17, 2018
++ Athlete-specific data requires authentication.
```
Comment (zackisaacs) [inline content/docs/changelog.md]: @MateoAO PTAL here.

**Pull request merged**